infrastructure/environments/dplplat01/sites.yaml

```diff
@@ -24,6 +24,7 @@ sites:
     releaseImageRepository: ghcr.io/danskernesdigitalebibliotek
     releaseImageName: dpl-cms-source
     dpl-cms-release: "2024.18.3"
+    importTranslationsCron: "0 * * * *"
```

Comment:
Maybe the default translation import cron schedules should be defined everywhere in the sites yaml file - also on the std sites.
I am aware of the defaults [here](https://github.com/danskernesdigitalebibliotek/dpl-platform/pull/303/files#diff-52b177fa1883f37996c67a5555015c80688f680bdfe28462b76d38e98f6c81c3R120-R123). But having them defined in sites.yaml makes it more visible what the differences are in the settings - I think.

Reply:
It's a good point, and easy to implement (non-breaking) if we want to at some point in the future. I'll leave it for now but keep it in mind for future changes to sites.yaml.

As we start having regular releases on several different plans etc we'll learn what makes sense in sites.yaml, and how verbose we want to be :-)